pallets/schemas/src/benchmarking.rs

```diff
@@ -1,40 +1,36 @@
-use frame_benchmarking::{benchmarks, whitelisted_caller};
+use frame_benchmarking::{benchmarks, vec, whitelisted_caller};
 use frame_support::{assert_ok, ensure};
 use frame_system::RawOrigin;
-use sp_std::vec::Vec;
 
 use crate::Pallet as SchemasPallet;
 
 use super::*;
 
-const SCHEMAS: u32 = 5000;
+const SCHEMAS: u32 = 1000;
 
-const EXAMPLE_SCHEMA: &[u8] = r#"{"type":"record","name":"mrc.schema.root",
```

Comment:
removing since we have no use yet and we will need to generate schemas in a different way after adding schema on-chain checks